x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/AbstractFunctionTestCase.java

```diff
@@ -256,8 +258,8 @@ private void testEvaluate(boolean readFloating) {
     /**
      * Evaluates a {@link Block} of values, all copied from the input pattern, read directly from the page.
      * <p>
-     *     Note that this'll sometimes be a {@link Vector} of values if the
-     *     input pattern contained only a single value.
+     * Note that this'll sometimes be a {@link Vector} of values if the
+     * input pattern contained only a single value.
```

Comment:
I think these came from my running the formatter. I'm confused how it wasn't committed before, but whatever.